plugins/obs-ffmpeg/obs-ffmpeg-mux.c

```diff
@@ -45,6 +45,7 @@ struct ffmpeg_muxer {
 	int64_t stop_ts;
 	uint64_t total_bytes;
 	struct dstr path;
+	const char *printable_path;
```

Comment:
Also add a comment about printable_path: It's the path with stream key information removed.